src/SiteConfig/ConfigBuilder.php

```diff
@@ -341,6 +345,8 @@ public function parseLines(array $lines)
             } elseif ((substr($command, -1) == ')') && preg_match('!^([a-z0-9_]+)\((.*?)\)$!i', $command, $match) && $match[1] == 'replace_string') {
                 array_push($config->find_string, $match[2]);
                 array_push($config->replace_string, $val);
+            } elseif ((substr($command, -1) == ')') && preg_match('!^([a-z0-9_]+)\(([a-z0-9_-]+)\)$!i', $command, $match) && $match[1] == 'http_header' && in_array($match[2], array('user-agent'))) {
```

Comment:
Instead of `in_array($match[2], array('user-agent'))` can you use a simple `'user-agent' === $match[2]`?